packages/tools/devtools/devtools-view/package.json

```diff
@@ -81,6 +82,7 @@
 	},
 	"devDependencies": {
 		"@arethetypeswrong/cli": "^0.15.2",
+		"@axe-core/react": "^4.9.1",
```

Comment:
Similar to jest-axe, I'm not entirely sure if we're using this one.

Also slight concern that its docs say "This package does not support React 18 and above", so it would limit us from upgrading from React 17.